### DISCUSSION TITLE
Hi all, Lourens from Durham in UK (originally from Cape Town South Africa)

### DISCUSSION THREAD

**Discussion by LourensWalters** [OpenMined/courses#55] [Introduce Yourself]
Really pleased to meet all of you! I can see you have a lot of experience in the field of Privacy, Security and Society. Looking forward to engaging on these interesting and relevant topics during the course. 

I worked as a Data Scientist in the healthcare financing industry in both public and private spheres for the past 14 years in both an analytic/ developer and management role. I am currently working for a Scientific Technology Innovation Accelerator in the NE of the UK. 

I have come up against many hurdles regarding protection of private information in my career. To date I mainly resolved them with the help of lawyers :) (which didn't always result in the best outcomes)! I have also been involved in Healthcare Information Exchanges (HIEs) and Electronic Health Record initiatives at a National Level in South Africa. I was chairperson of the Private Health Information Standards Committee (PHISC) EHR subcommittee in South Africa for 3 years. I am therefore acutely aware of the value of information sharing as well as the sensitive nature thereof.  

Before working as a data scientist I was a software engineer developing Electronic Fund Transfer (EFT) software. 

I am interested in learning more about technical solutions to privacy of information issues (as well as gaining trust and working with others towards collective goals).

**Comment by iamtrask** [comment]
Hi Lourens - great to meet you too! I'll be super curious to hear about your thoughts having battled private data challenges before. There aren't that many folks who have the experiences you describe - I imagine we'll learn a lot from being able to chat with you around this discussion board and on slack. What kinds of legal solutions did you employ?

**Reply by LourensWalters** [reply]
Hi Andrew, probably not the kind of solutions you are attempting i.e. holistic solutions to the problem at large. In the main I set up legal information/ data sharing agreements with other entities such as Memoranda of Understanding (MOUs) and Non-Disclosure Agreements (NDAs). Our company had strict guidelines as to whom we may share information with and under what conditions (mainly academic institutions, and only within specified research criteria). These conditions were so restrictive that a lot of potential value from the information was lost. The process was also very onerous and precluded the sharing of information save for a few cases that were financially viable - this was pre GDPR (POPI in South Africa), so I guess now it will be even more difficult to share information.

The data was typically de-identified prior to sharing (Safe Harbour criteria typically used). This was a very onerous and manual process - and quite error prone. 

The work on HIEs I was involved with was much more promising as it involved a technical solution too. But this was longer term solution for the organisations involved 3 - 5 years to build and deploy. It also cost a lot and the issue of trust between competitor organisations was a huge hurdle, so I don't know if it ever saw the light. I unfortunately left the company after the 1st year of the project so don't know the outcome. Great idea though!
